examples/viewer_fps_label.py

```diff
@@ -21,7 +21,7 @@ def update_fps(fps):
 viewer.text_overlay.visible = True
 # note: this is using a private attribute, so it might break
 # without warningin future versions!
-viewer.window._qt_viewer.canvas._scene_canvas.measure_fps(callback=update_fps)
+viewer.measure_fps(callback=update_fps)
```

Comment:
The above comment does not look relevant now.

Reply:
true, though before changing this what is your opinion on keeping the example as is? The problem of the user was most likely due to having the newer version of napari, but old example. If everyone agrees to postpone fixing I will just close this PR.

Reply:
yes. Please just point new private path instead of this fix

Reply:
ok will do it for now, though I do believe that this function could be an entrypoint to all canvases if a user wants to measure fps on all, but will close for now.